### ISSUE TITLE
Fix QuickSight Notifications import failure

### ISSUE THREAD

**Issue by sastels** [cds-snc/notification-planning-core#386]
## Describe the bug 

The production `notifications` dataset in QuickSight is failing during its nightly creation with the error:
```
ERROR: could not write to file "base/pgsql_tmp/pgsql_tmp12782.30": No space left on device
```

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
See examples in the documentation

SEV-2 Major: QuickSight will only contain notifications data older than July 16.

## To Reproduce

View the nightly failures in production at
https://ca-central-1.quicksight.aws.amazon.com/sn/data-sets/notifications/view

## Expected behavior

Dataset created with up to date data.

## Impact

CDS can no longer rely on QuickSight for data analytics related to new notifications.

## Additional context

[See this Slack thread.](https://gcdigital.slack.com/archives/CV38DBNVA/p1721227476009529)

TL;DR the joins of the full notification_history table with services, templates, etc are causing the RDS instances to run out of local storage.

Options for fixing:

1. Increase the size of the database instances. Note that this may fix the problem currently, but as the history table grows the issue will reoccur. 
2. Move the history table (and perhaps other data) to another database / data lake with a different configuration that would allow the QuickSight import to succeed.
3. Save notifications to the history table every night rather than as required by data retention. Then we could read just the history table into QuickSight had (hopefully) perform the joins in QuickSight / SPICE. rather than in RDS.
4. Create a Python task to incrementally build the dataset in RDS and import this new table into QuickSight.

**Comment by sastels** [comment]
This was fixed by boosting the class of the RDS instances
